src/utils/misc/index.ts

```diff
@@ -0,0 +1,22 @@
+export const formDataToJSON = <T>(formData: FormData) => {
```

Comment:
This is VERY bad, but I did not found a solid way to convert formData to JSON object.

Reply:
I removed this function in favor of a just as bad custom form data extraction https://github.com/indigo-iam/iam-dashboard/blob/f64b5e1f5a416ad5e8da5bf09bfdcbbb5b21104d/src/services/clients/index.ts#L61

Reply:
@SteDev2 Can you have a look please?

Reply:
this solution is good for me